src/test/java/teammates/test/cases/ui/browsertests/InstructorCoursesPageUiTest.java

```diff
@@ -323,6 +323,7 @@ public void testArchiveAction() throws Exception {
         coursesPage = getCoursesPage();
             
         coursesPage.unarchiveCourse(courseId);
+        coursesPage.waitForAjaxLoadCoursesSuccess();
```

Comment:
If this (waiting after un-archiving a course) is needed, then shouldn't we wait after clicking on archive too?